cmd/mdmctl/apply.go

```diff
@@ -56,6 +60,7 @@ Apply a resource.
 Valid resource types:
 
   * blueprints
+  * dep-tokens
```

Comment:
All that said any objection to finishing up the PR, merging and we can refactor later?